src/olympia/activity/tests/test_models.py

```diff
@@ -99,6 +99,129 @@ def test_rejected_version_still_valid(self):
         assert self.token.is_valid()
 
 
+class TestActivityLogManager(TestCase):
+    def test_pending_for_developer(self):
+        to_create = (
```

Comment:
curious why not using a parameterized test here? Again seems to predate but this would parallelize the tests and let each fail independently of the others.

Reply:
`@pytest.mark.parametrize` is not really usable with `TestCase` - it wants to feed all the arguments to the function, so that doesn't play well with methods that should receive `self` as the first argument.

Reply:
Right. In those cases I've just used functional tests. Just wondering as I've used it before and wonder if we have a stigma against it or not.

Reply:
I don't love `@parametrize` because of that limitation (and general awkwardness around how arguments are defined when using it) but nobody would stop you if you were to use it in new code. We do have several already.